src/main/java/com/synopsys/integration/alert/provider/polaris/PolarisProvider.java

```diff
@@ -45,19 +47,24 @@
 
     private final TaskManager taskManager;
     private final PolarisProjectSyncTask projectSyncTask;
+    private final PolarisProperties polarisProperties;
 
     @Autowired
-    public PolarisProvider(final TaskManager taskManager, final PolarisProjectSyncTask projectSyncTask) {
+    public PolarisProvider(final TaskManager taskManager, final PolarisProjectSyncTask projectSyncTask, final PolarisProperties polarisProperties) {
         super(PolarisProvider.COMPONENT_NAME, null);
         this.taskManager = taskManager;
         this.projectSyncTask = projectSyncTask;
+        this.polarisProperties = polarisProperties;
     }
 
     @Override
     public void initialize() {
         logger.info("Initializing Polaris provider...");
         taskManager.registerTask(projectSyncTask);
-        taskManager.scheduleCronTask(BlackDuckAccumulator.DEFAULT_CRON_EXPRESSION, projectSyncTask.getTaskName());
+        final Optional<AccessTokenPolarisHttpClient> polarisHttpClient = polarisProperties.createPolarisHttpClientSafely(logger);
+        polarisHttpClient.ifPresent(client -> {
+            taskManager.scheduleCronTask(BlackDuckAccumulator.DEFAULT_CRON_EXPRESSION, projectSyncTask.getTaskName());
```

Comment:
Since this sort of functionality seems to be appearing multiple times, maybe we want to build it into the provider or make a new parent Provider called ScheduledProvider (Or something similar). The provider will have all the same methods available, but also only start the job if a config exists.